lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.h

```diff
@@ -269,6 +269,14 @@ class ObjectFileELF : public lldb_private::ObjectFile {
 
   lldb::SectionType GetSectionType(const ELFSectionHeaderInfo &H) const;
 
+  // Parse and ELF header section to detect the RISCV attributes.
+  // This function traverses the section headers of the ELF object file to
+  // verify the presence of the Xqci feature extension in the RISCV attributes.
+  // If the extension is found, it updates the '+xqci' feature string.
```

Comment:
I get that this function may only support that one extension name, but unless you're gonna call this function `ParseRISCVAttributesButOnlyLookForxqci`, it should not be documented like this.

If you want to note that it will only find features LLDB has been made aware of, that would be nice.